Frontend/src/components/teams/overview/TeamCreationDialog/index.jsx

```diff
@@ -0,0 +1,71 @@
+/* eslint-disable no-return-assign */
+import React, { useState } from 'react';
+import Button from '@mui/material/Button';
+import LoadingButton from '@mui/lab/LoadingButton';
+import TextField from '@mui/material/TextField';
+import Dialog from '@mui/material/Dialog';
+import DialogActions from '@mui/material/DialogActions';
+import DialogContent from '@mui/material/DialogContent';
+import DialogTitle from '@mui/material/DialogTitle';
+import { createTeam } from 'shared/services/mock/teams';
+
+export default function TeamCreationDialog({ open, handleClose, onCreated }) {
+  const [name, setName] = useState('');
+  const [description, setDescription] = useState('');
+  const [loading, setLoading] = useState(false);
+  const [nameError, setNameError] = useState('');
+  const [descriptionError, setDescriptionError] = useState('');
+
+  async function create() {
+    if (!name) {
+      setNameError('Please enter a name.');

```

Comment:
If we need in a lot of cases to validate the input, we might need a validator library. But for now it is fine.